index.html

```diff
@@ -7,13 +7,103 @@
+                        <img src="images/picText4.jpg" alt="heroPicture" width="100%">
+                        <div class="img-text row">
+                            <h6 class="col-12 h-center">AMBITON IS THE FIRST STEP TOWARDS</h6>
+                            <h1 class="col-12 h-center">SUCCESS</h1>
+                            
+                                <button class="col-1"><h6>BOOK NOW</h6></button>
+                            
+                       </div>
+                    </div>
 
-    <!-- Add your markup here -->
+            
+            </section>
+
+            <section class="col-12 row info-section">
+                <div class="col-6 row about-me">
+                    <h1 class="col-12 h-about">ABOUT</h1>
+                    <h1 class="col-12 h-me">ME</h1>
+                    <div class="col-12 about-me">
+                      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio debitis ipsa nesciunt alias voluptatum veniam velit quidem sit aliquam inventore repellendus, ab iste? Adipisci atque delectus iure accusamus eveniet minima.</p>
+                    </div>
+                    <button class="col-3 h-button">
+                        <h4>Read More</h4>
+                    </button>                    
+                </div>
+                <div class="col-6 info-pic">
+                   <img src="images/picText4.jpg" alt="imagCompany">
+                </div>
+            </section>
+            <section>
+                <div class="col-12 row">
+                    <div class="col-2"></div>
+                    <div class="col-8">
+                        <h1>HOW I CAN</h1>
+                        <h1>HELP YOU</h1>
+                        <div class="box-boarder">
+                            <h3>ONE ON ONE COACHING</h3>
+                            <p>I am a tagline.Click here to add your own text and edit me</p>
+                            <hr>
+                            <p>1 hr</p>
+                            <p>$70</p>
+                            <div>
+                              <button class="btn">Book it</button>
+                           </div>
+                        </div>
+                        <div class="btn-wraper row">
+                           <button class="btn">More services</button>
+                        </div>
+                    </div>
+                    <div class="col-2"></div>
+                </div>
+            </section>
+        </main>
+           <footer class="row">
+               <div class="contact col-6">
+                   <div class="col-2"></div>
+                   <div class="col-8"><h3>CONTACT</h3>
+                    <P>67 Terryfranco Road</P>
+                    <p>Sanfrancisco SF156S</p>
+                </div>
+                   <div class="col-2"></div>
+               </div>
+               <div class="form col-6">
+                 <div>
+                    <label class="label">Enter Your Name</label>
```

Comment:
The `label` and `input` classes don't seem to exist in your CSS. You don't need these.